packages/gitlab/dev/index.tsx

```diff
@@ -41,6 +42,17 @@ createDevApp()
             return {
                 build: () => cli,
             };
+
+            // Here we mock the client requests to GitLab
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
```

Comment:
This code should be put before the return statement.

Reply:
Fixed. Also fixed eslint errors in tests file